kappa: 5
sigma_1: 0.7
sigma_2: 0.2
sigma_3: 0.157
p: 2
c: 1.0
l: 0.1
theta_0: 1.57
phi_term: 0.0
mu_1: 1.0
mu_2: 1.0
rho: 0.7
gamma: 1
large: 100.0
delta: 10.0
epsilon_1: 0.3
epsilon_2: 0.3
use_sim_time: false
barrier_field_size: 4.0
barrier_field_points: 10